physical/cassandra/cassandra.go

```diff
@@ -217,8 +217,10 @@ func setupCassandraTLS(conf map[string]string, cluster *gocql.ClusterConfig) err
 			tlsConfig.MinVersion = tls.VersionTLS11
 		case "tls12":
 			tlsConfig.MinVersion = tls.VersionTLS12
+		case "tls13":
+			tlsConfig.MinVersion = tls.VersionTLS13
 		default:
-			return fmt.Errorf("'tls_min_version' must be one of `tls10`, `tls11` or `tls12`")
+			return fmt.Errorf("'tls_min_version' must be one of `tls10`, `tls11`, `tls12` or `tls13`")
```

Comment:
Does Cassandra support TLS 1.3? I see an issue [here](https://issues.apache.org/jira/browse/CASSANDRA-15048) from a while ago, but it's still open. I haven't found it explicitly mentioned in their docs. The current test coverage of this code doesn't exercise TLS. Might be a good idea to add test coverage in `cassandra_test.go` to verify that it does work.